### DISCUSSION TITLE
Resuming after launching a window form from a usercontrol

### DISCUSSION THREAD

**Discussion by chasihle** [canton7/Stylet#194] [Q&A] [answered]
Hi,
I have a situation where I launch a window form from a usercontrol. After closing the window I need to reload a datagrid. You provided the code below to do that when coming back to a usercontrol from another user control but this doesn't seem to work for coming back to a usercontrol from a window. Is there a way to do this?

this.Bind(s => s.ActiveItem, (o, e) =>
            {
                if (e.NewValue == null)
                {
                    // do something
                }
            });

thanks

**Comment by canton7** [comment, marked as answer]
Call the reload method after ShowDialog returns

**Reply by chasihle** [reply]
That worked. I was using Show.Window instead of Show.Diolag
